.stylelintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "scss/at-rule-no-unknown": true,
+      "csstree/validator": true
+    },
+    "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+  }
```

Comment:
Great code you have up there Goodluck5956 👋👋, However, I'll suggest you add a newline after the file because a file that is not empty and does not end with a newline is therefore not a text file. kindly fix that and re-submit for review.